main/building_blocks/logical/LogicalOperator.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, paulis: List[Pauli]):
         # Shouldn't access _paulis directly; instead, use at_round.
         self._paulis = paulis
 
-        self._assert_non_empty()
```

Comment:
I'm doing this so I can use class inheritance for timelike logical operators.

Reply:
I'm wondering if I should do this. A stability observable _is not_  a timelike logical operator.